Editing and proofreading of hunting.md

lupomikti opened the pull request:
Taking advantage of the ability to suggest changes in this manner to do a bit of an overhaul on the quality of the guide here. The motivation was simply that I was trying to follow along with it and had a frustratingly difficult time comprehending some of what it wanted me to do. I may have worked my through it but if I was having trouble I'm sure many others have even more trouble.

Thus I'm putting my experience with documentation writing from my job to good use here (I have to create guides and explanations on semi-technical processes for non-technical users a lot).

Also of note:
- `enable-hunting.png` is now out-of-date as XXMI's settings interface has changed. This image should be updated if possible.
- similarly, gui_collect just had its 1.0.0 release and there are some slight differences in screenshots now (mostly additions, not changes)

My goal is to slowly do this for each guide/file here as I find I have similar trouble following along. Although some of those are going to be a while yet as I don't have the experience/knowledge for what they cover and cannot meaningfully improve what's written.

Edit: I think I missed a space on line 89 in the changed file between `:::` and `warning`

lupomikti committed:
Editing and proofreading of hunting.md

Extensive editing and rewriting for clarity to meet a standard of excellence for documentation and guide writing.

leotorrez commented:
The updated image was provided in https://github.com/leotorrez/modding/pull/5, I will add it to the English version as well when I merge the RU translation.

lupomikti committed:
hunting.md proofreading

Did another proofreading pass to correct any small typos or omissions that were left
